### PR TITLE
Update acronyms.md

### PR DESCRIPTION
**Pull request by suprenant**

### CODE REVIEW

File: acronyms.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ This is a guide of commonly used acronyms that you may encounter in this engagem
 
 **AAP** Advance Analytics Package
 
-**ADCS** Automated Data Collection System -- Used in repair process to collect data on aircraft, engines and parts
+**ADCS** Automated Data Collection System - Used in repair process to collect data on aircraft, engines and parts
 
 **ASW** Anti-submarine warfare
 
+**BPR** Business process reengineering
```
Comment (tbaxter-18f) [inline acronyms.md]: I think it should probably be hyphenated -- re-engineering

File: acronyms.md [hunk 2]
```diff
@@ -18,22 +20,40 @@ This is a guide of commonly used acronyms that you may encounter in this engagem
 
 **DCPDS** Defense Civilian Personnel Data System
 
-**DIFMS** Defense Industrial Financial Management System
+**DIFMS** Defense Industrial Financial Management System - named financial system
```
Comment (tbaxter-18f) [inline acronyms.md:23]: No "DEPO"?  :-D

File: acronyms.md [hunk 3]
```diff
@@ -18,22 +20,40 @@ This is a guide of commonly used acronyms that you may encounter in this engagem
...
+**IPT** Integrated Product Team - work team on the floor, a group of artisans
+
+**LER** Labor & Employee Relations
+
+**MRO** Maintenance, Repair, and Overhaul - Often referred to as MRO Process
 
-**NAS Pax River** Naval Air Station Patuxent River -- In St. Mary’s County, Maryland, is home to NAVAIR Headquarters, affiliated Program Executive Officers, and the Naval Air Warfare Center Aircraft Division
+**NAE** Naval Aviation Enterprise
+
+**NAS Pax River** Naval Air Station Patuxent River - In St. Mary’s County, Maryland, is home to NAVAIR Headquarters, affiliated Program Executive Officers, and the Naval Air Warfare Center Aircraft Division
```
Comment (tbaxter-18f) [inline acronyms.md:41]: Extra line above?

File: acronyms.md (hunk 2 above)
Comment (suprenant) [reply acronyms.md:23]: lol

**Pull request merged**